docs/testplan/PFC_Snappi_Additional_Testcases.md

```diff
@@ -0,0 +1,65 @@
+|           07 | CONGESTION (MACSEC)      | **CONGESTION OVER MISMATCHED INGRESS AND EGRESS MACSEC ENABLED LINK:**<br>Test case to check if the DUT detects congestion due to mismatched MACSEC enabled link speeds and sends PFCs to rate-limit the lossless priority traffic.                                                                                       | - Single 400Gbps Ingress and single 100Gbps egress <br> - Repeat with IPv6 <br>- 1024B packet-size.                                                                                                                                    | - 90% - ingress line rate Priority 3 and 4 <br>- 10% of ingress line-rate -- Background Flow[Priority 0, 1, and 2]                                                                                                                                                     | - Single LC - Multiple ASIC <br>- Multiple LC                                    | 1. There should be NO packet drops on the DUT for both Lossless and lossy priority traffic.<br>2. DUT should send PFCs to the traffic generator to rate-limit the lossless priority traffic.<br>3. 100Gbps Egress port should receive Lossless Priority 3 and 4 traffic.                                                                                                                                                                                                                       |
+|           08 | CONGESTION (PC)          | **UNEQUAL SPEED INGRESS-EGRESS PORTCHANNEL CONGESTION TEST:**<br>Test case to check that DUT detects congestion due to mismatched ingress and egress port-channel speeds and send PFCs to the ingress to rate-limit the lossless priority traffic. No drops for the lossless priority traffic.                            | - Port-channel - N x 100Gbps Ingress - 400Gbps Egress with N > 4. <br>- 1024B packet-size.                                                                                                                                             | - 90% of total ingress port-channel link- Priority 3 and 4 <br>- 10% of ingress port-channel link- Background Flow[Prio0, 1, and 2]                                                                                                                                    | - Single LC - Multiple ASIC <br>- Multiple LC                                    | 1. The lossless Prio3-4 traffic should not experience drop.<br>2. Ensure that background flows experience drop.<br>3. DUT sends PFCs to traffic generator to rate-limit the lossless prio traffic.                                                                                                                                                                                                                                                                                             |

```

Comment:
should this test be flexible to use 2x400G PC in and 1x400G out?
likewise: 2x100G in and 1x100G out